enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationStatusService.java

```diff
@@ -194,4 +197,21 @@ private void evaluatePresenceCheckResult(OwnerId ownerId,
         }
     }
 
+    /**
+     *
+     *
+     * @param ownerId Owner identification.
+     * @param idVerification Verification identity.
+     */
+    private void startVerification(OwnerId ownerId, IdentityVerificationEntity idVerification) {
+        try {
+            identityVerificationService.startVerification(ownerId);
+        } catch (DocumentVerificationException e) {
```

Comment:
We'll need to merge latest changes from in error handling:
https://github.com/wultra/enrollment-server/pull/134/files#diff-dbc4228c7aa1cf01460a26f414926790d475030a5497e0a03353a2ba36661c1dR179

Reply:
resolved